### ISSUE TITLE
v2.4 of WebTools UnsupportedAppStore Error

### ISSUE THREAD

**Issue by BenjaminHCCarr** [ukdtom/WebTools.bundle#264]
When I run the check for update script `Check for updates for all installed channels`

I get:
---
# An error occured.
## Error Code/Message: 500/Internal Server Error

---
I newly migrated from v2.3 I am on:
`2017-02-15 22:29:44,778 (802406400) : INFO (core:349) - Starting framework core - Version: 2.6.3, Build: b46e0ea (Wed Feb 01 15:54:52 UTC 2017)`

based on digging through the the logs it looks like it is the `VH1.bundle` which is calling out to:
```
#486	2017-02-16 01:08:44,179 (80a116400) : DEBUG (networking:166) - Requesting '(https://github.com/shopgirl284/VH1.bundle/commits/.atom)'
#487	2017-02-16 01:08:44,701 (80a116400) : ERROR (networking:219) - Error opening URL '(https://github.com/shopgirl284/VH1.bundle/commits/.atom)'
#488	2017-02-16 01:08:44,703 (80a116400) : CRITICAL (git:270) - Fatal error happened in getUpdateList: HTTP Error 404: Not Found (most recent call last):
```
That entire repo seems to be yanked: (https://github.com/shopgirl284/VH1.bundle/)

Removing the `VH1.bundle` allows checking for updates to continue, and produces 5 bundles with updates.

It appears that any 404s in the bundles halts the code at that point, rather than warning and continuing.

**Comment by ukdtom** [comment]
Heh, and a bug.

Try this:

From the menu of WebTools, select "Factory Reset"
Then after it's done, try the migrate again...

Does that work?

/T

**Comment by BenjaminHCCarr** [comment]
I can test, but I am 99.87% sure it will pass, as removing the dead repo `vh1.bundle` allowed it to not not `404` and offer the button to `update all`.

**Comment by BenjaminHCCarr** [comment]
yes, running:
`Factory Reset` I get a list of:
`Below is the list of migrated bundles. These should be managed via Webtools from now on.`

And the `Massive Updater` which has already passed after removing `vh1.bundle passed again, this time with one update to `Plex Request Channel`

Logs attached.

I think the greater concern is the effective halt on timeout/404 of a single bundle, this could be troublesome in-particular with non-standard/unknown plugins.

[com.plexapp.plugins.WebTools.log.1.txt](https://github.com/ukdtom/WebTools.bundle/files/779981/com.plexapp.plugins.WebTools.log.1.txt)
[com.plexapp.plugins.WebTools.log.txt](https://github.com/ukdtom/WebTools.bundle/files/779980/com.plexapp.plugins.WebTools.log.txt)
[com.plexapp.plugins.WebTools.log_after_Reset.txt](https://github.com/ukdtom/WebTools.bundle/files/779982/com.plexapp.plugins.WebTools.log_after_Reset.txt)

**Comment by ukdtom** [comment]
Will be fixed in 2.4.1
